SetupGuide_Plugin_Admin.php

```diff
@@ -747,7 +747,7 @@ public function config_browsercache() {
 	 * @see \W3TC\Config::get_array()
 	 */
 	public function get_lazyload_settings() {
-		if ( wp_verify_nonce( $_POST['_wpnonce'], 'w3tc_wizard' ) ) { // phpcs:ignore WordPress.Security.ValidatedSanitizedInput
+		if ( wp_verify_nonce( Util_Request::get_string( '_wpnonce' ), 'w3tc_wizard' ) ) { // phpcs:ignore WordPress.Security.ValidatedSanitizedInput
```

Comment:
Remove the phpcs comment to have it validate the change.

Reply:
Removed PHPCS disable comment.